docs/content/latest/api/ycql/expr_fcall.md

```diff
@@ -87,7 +88,34 @@ CAST function converts the value returned from a table column to the specified d
 | `TIMESTAMP` | `DATE`, `TEXT` |
 | `TIMEUUID` | `DATE`, `TIMESTAMP` |
 
-### Examples
+## WriteTime function
+We can get the timestamp when the column/row was writen.
+Assuming we have a table `page_views` and a column named `views`:
+```
+ SELECT writetime(views) FROM page_views;
+
+ writetime(views)
+------------------
+ 1572882871160113
+
+(1 rows)
+```
+
+## TTL function
+We can get the number of seconds until a column or row expires. 
```

Comment:
Change wording style from:

We can get the 

to:

The TTL function returns ...